src/test/server/database/test_database.py

```diff
@@ -243,6 +243,14 @@ def test_get_work_machine(self) -> None:
         self.mockDatabase.update_work_machine(work_machine)
         self.assertEqual(self.mockDatabase.get_work_machines(), [self.work_machine])
 
+    def test_bug_found_1(self) -> None:
```

Comment:
Can we not give it a better name? Like `test_offline_machines` ? Or at least name it after the issue on Github that this tests.